crnk-client/src/main/java/io/crnk/client/internal/ResourceRepositoryStubImpl.java

```diff
@@ -48,8 +51,26 @@ public ResourceRepositoryStubImpl(CrnkClient client, Class<T> resourceClass, Res
     }
 
     @Override
-    public void delete(List<I> id) {
-        throw new UnsupportedOperationException("not yet implemented");
+    public void delete(List<I> ids) {
+        String url = urlBuilder.buildUrl(resourceInformation, null, (QuerySpec) null);
+
+        final ObjectMapper objectMapper = client.getObjectMapper();
+
+        HashMap requestDocument = new HashMap();
+        requestDocument.put("data", ids.stream().map(id -> {
```

Comment:
you could use here the Crnk "Document" class and fill it up with ResourceIdentifier. This way everything remains slightly more type-safe. ResourceInformation further has methods to serialize an ID to String and back, which become important for more complex ID types.